site/docs-md/guides/push-notifications-firebase.md

```diff
@@ -0,0 +1,369 @@
+
+```
+implementation 'com.google.firebase:firebase-messaging:17.4.0'
+```
+
+**Note**: Make sure that the file is added to source control. You may need to modify the `.gitignore` file under `android/` to add the file.
+
+## iOS
+
+### Prerequisites
+
+iOS push notifications are significantly more complicated to set up than Android. You must have a [paid Apple Developer account](https://developer.apple.com/) *and* take care of the following items prior to being able to test push notifications with your iOS application:
+
+1. [Setup the proper Development or Production certificates & provisioning profiles](https://help.apple.com/xcode/mac/current/#/dev60b6fbbc7) for your iOS application in the Apple Developer Portal
+2. [Create an APNS certificate or key](https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/establishing_a_certificate-based_connection_to_apns) for either Development or Production in the Apple Developer Portal
+3. [Ensure Push Notification capabilities have been enabled](https://help.apple.com/xcode/mac/current/#/dev88ff319e7) in your application in Xcode
+4. Have a physical iOS device running one of the two latest iOS versions (push notification functionality does not work in the iOS simulators)
+
+### Integrating Firebase with our native iOS app
+
+This part is very similar to the Android section above, with a few key differences.
+
+First, go to the **Project Overview** page for your Firebase project. If you've been following this guide, you'll already have an Android application listed at the top of the page.
+
+To add iOS to your Firebase project, click the **Add App** button and select the **iOS** platform.
+
+The next screen will ask you for some information about your application.
+
+- Your **iOS bundle ID** should match the **appId** from your `capacitor.config.json` file
+- We used `com.mydomain.myappname` for this Capacitor app ID, so that is what we'll use for this entry.
+- App Nickname and App Store ID are optional
+
+Then click the **Register app** button.
+
+### Add the `GoogleService-Info.plist` file to your iOS app
+
+**Note**: This is **not** the same file used for your Android app.
+
+Download the `GoogleService-Info.plist` provided to your local machine and move it into your Xcode project as instructed, ensuring to add it to all targets. 
+
+![Google Service Info Plist Location for iOS](/assets/img/docs/guides/firebase-push-notifications/google-plist-location-ios.png)
+
+**Note**: It is recommended to use Xcode to move the `.plist` file into your project.
+
+### Add the Firebase SDK via CocoaPods
+
+The Push Notification API on iOS makes use of CocoaPods - an iOS dependency management system - and we need to tell CocoaPods to make use of Firebase.
+
+To do this, we need to modify the `Podfile`, which is located in your `ios/App` directory:
```

Comment:
The Podfile is accessible from the Xcode project, probably better to open and edit it from there